src/bioregistry/data/bioregistry.json

```diff
@@ -70498,6 +70498,35 @@
       }
     ]
   },
+  "mirgenedb": {
+    "contact": {
+      "email": "BastianFromm@gmail.com",
+      "name": "Bastian Fromm",
+      "orcid": "0000-0003-0352-3037"
+    },
+    "contributor": {
+      "email": "palayoor.n@northeastern.edu",
+      "github": "nalikapalayoor",
+      "name": "Nalika Palayoor",
+      "orcid": "0009-0008-8406-631X"
+    },
+    "description": "mircoRNA Gene Database is a collection of animal microRNA genes. This database allows researchers to analyze mircoRNA properties like secondary structure and features in a phylogenetic context.",
+    "example": "hsa/Let-7-P1b",
+    "github_request_issue": 1393,
+    "homepage": "https://mirgenedb.org/",
+    "name": "microRNA Gene Database",
+    "pattern": "^[a-zA-Z0-9-]+/[a-zA-Z0-9-]+$",
+    "publications": [
+      {
+        "doi": "10.1093/nar/gkae1094",
+        "pmc": "PMC11701709",
+        "pubmed": "39673268",
+        "title": "MirGeneDB 3.0: improved taxonomic sampling, uniform nomenclature of novel conserved microRNA families and updated covariance models",
+        "year": 2025
+      }
+    ],
+    "uri_format": "https://mirgenedb.org/show/$1"
```

Comment:
There are multiple prior publications about MirGeneDB that we should curate here:

 MirGeneDB is a database of manually curated microRNA genes that have been validated and annotated as initially described in [Fromm et al. 2015 ](http://www.annualreviews.org/doi/abs/10.1146/annurev-genet-120213-092023), [Fromm et al. 2020](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz885/5584683?guestAccessKey=b9fe4339-a36b-49df-96ea-9e2cd7a1c99b) and [Fromm et al 2022](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkab1101/6439665). MirGeneDB 3.0 ([Clarke and Hoye et al. 2024 ](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkae1094/7914206?login=true))